Assignment1/irisdata.py

```diff
@@ -0,0 +1,246 @@
+import sys
+
+import numpy as np
+import pandas as pd
+import plotly.express as px
+
+
+def main():
+    # Loading the dataset
+    data_df = pd.read_csv("Iris.csv")
+    print(data_df.head())
+    print(data_df.tail())
+    # Getting simple summary stats using pandas
+    print(data_df.describe())
+    # checking the columns of data
+    print(data_df.columns)
+    # checking the count of species
+    print(data_df["species"].value_counts())
+    # converting the data to numpy array and getting summary
+    x = data_df.iloc[:, 0:4].values
+    # summary of the data
+    print(
+        "Sepal Length Summary [",
+        "Mean :",
+        np.mean(x[:, 0]),
+        "Median :",
+        np.median(x[:, 0], axis=None),
+        "Minimum:",
+        np.min(x[:, 0], axis=None),
+        "Maximum :",
+        np.max(x[:, 0], axis=None),
+        "1st quartile:",
+        np.quantile(x[:, 0], 0.25, axis=None),
+        "3rd Quartile",
+        np.quantile(x[:, 0], 0.75, axis=None),
+    )
+    print(
```

Comment:
I think you should create a function or use a loop to generate statistics and print them. It will remove the code duplication and be the more optimal way to implement it.

Reply:
Rodger that!